pkgs/development/libraries/qtforkawesome/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cpp-utilities
+, qtutilities
+, qttools
+, qtbase
+, cmake
+, perl
+}:
+
+let
+  fork_awesome_release = fetchFromGitHub {
+    owner = "ForkAwesome";
+    repo = "Fork-Awesome";
+    rev = "1.2.0";
+    sha256 = "sha256-zG6/0dWjU7/y/oDZuSEv+54Mchng64LVyV8bluskYzc=";
+  };
+in stdenv.mkDerivation rec {
+  pname = "qtforkawesome";
+  version = "0.0.4";
+
+  src = fetchFromGitHub {
+    owner = "Martchus";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-J3F+ikz6gQqV+JsOX8EpMOWoTmI6UK5KndFALUo4oiU=";
+  };
+
+  buildInputs = [
+    qtbase
+    cpp-utilities
+    qtutilities
+  ];
+  nativeBuildInputs = [
+    cmake
+    qttools
+    perl
+    perl.pkgs.YAML
+  ];
+  cmakeFlags = [
+    "-DFORK_AWESOME_FONT_FILE=${fork_awesome_release}/fonts/forkawesome-webfont.woff2"
```

Comment:
In case you're missing icons, make sure freetype is configured with brotli support or use the ttf file. (I'm just saying because looking at https://github.com/NixOS/nixpkgs/blob/aadd312b18d4bb833adadd5cb0f49dbd83c9a344/pkgs/development/libraries/freetype/default.nix I'm not sure whether brotli is enabled.)

Reply:
> In case you're missing icons, make sure freetype is configured with brotli support or use the ttf file. (I'm just saying because looking at [`aadd312`/pkgs/development/libraries/freetype/default.nix](https://github.com/NixOS/nixpkgs/blob/aadd312b18d4bb833adadd5cb0f49dbd83c9a344/pkgs/development/libraries/freetype/default.nix) I'm not sure whether brotli is enabled.)

Thanks, I haven't noticed any difference in the app's appearance, so I thought everything was ok, until I saw the missing icons in `syncthingtray --windowed`.

Reply:
For reference, a follow-up PR is in https://github.com/NixOS/nixpkgs/pull/174929 .